### ISSUE TITLE
System Status - OFF not supported

### ISSUE THREAD

**Issue by AndyNew2** [kodebach/hacs-idm-heatpump#168]
Further simple issue, but more difficult to solve:

The status "Aus" "Komplett aus ohne Frostschutz" is not supported by the IDM Integration. Unfortunately the value for this mode is -1 which results in 0xFFFF due to unsigned integer format selected for enum values.
That would not cause issues, if 0xFFFF was selected for the "unavailable" value mark. Therefore there is a collision for this mark. 
I propose to remove the "unavailable" mark and work just with the value enum, which would result in "Unknown" instead of unavailable. I do not see any advantange of having "unavailable"...

Following files needs changing to support this:
const.py to add OFF: 655535 for completely off for system state
sensor_addresses.py around line 277, remove the check for 0xFFFF here to allow the -1 value for OFF of SYSTEM STATUS
services.yaml add "OFF" as valid option for the set system status command
translations/de.json to add a meaning for OFF, like completely off --> komplett Aus
trnaslations/en.json to add a meaning for OFF, like completely off

If you need further help, I can upload the modified files, but I think it is easy to understand.

**Comment by kodebach** [comment]
We can't just remove the check for `0xFFFF` because that value is actually reported for things that are unavailable (e.g. heat circuits that aren't configured). I guess they reused the value here, because "System Status" can never be unavailable. I'll see what we can do...

**Comment by kodebach** [comment]
@AndyNew2 please verify that `v0.10.0` works for you. `0xFFFF` should now only be interpreted as "unavailable", if the Enum doesn't define a different name for the value.

**Comment by AndyNew2** [comment]
Yes will try and report. Sorry for the delay, was off a few days...

**Comment by AndyNew2** [comment]
Just tried v0.10.3 and state off works now. That could be set to fixed and issue could be closed. Thanks for support!
